integrations/zendesk/syncs/users.ts

```diff
@@ -58,7 +58,8 @@ const sync = createSync({
                     id: zUser.id.toString(),
                     firstName: firstName || '',
                     lastName: lastName || '',
-                    email: zUser.email
+                    email: zUser.email,
+                    user_fields: zUser.user_fields
```

Comment:
Thanks! You need to add this `user_fields` property and its shape to the `User` object in the [models](https://github.com/sohan2410/integration-templates/blob/e1587089870b5cbe144541581d268a886c04a878/integrations/zendesk/models.ts#L24-L28)